testRearrangeCarGroup8.py

```
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):

```

I think we should also test the case in which init_order and final_order are of different length - how our function handle that case? I believe it should throw an error of the sort "These are not the same car parks!" or "There is a mismatching in the car parks' sizes, please check them again". Something like that.

---

@blmussati I think this sounds like a very good idea! Do you think putting the check in the `__init__` function will be a good idea? Or should we put it in the method `rearrange_all_cars()`?

---

I temporarily put it in `rearrange_all_cars()`, but feel free to change it! :)

---

I cannot see the update but it might be because my git is still connecting to an old version, I'm not sure :/

---

I would putit in __init__ but since the deadline is passed I think rearrange_all_cars() will do :D